content/spin/kv-store-tutorial.md

```diff
@@ -433,12 +417,12 @@ Now let's build and deploy our Spin Application locally. Run the following comma
 $ spin build
 ```
 
-Now run the subsequent command to deploy your application: 
+Now, we have the option to add a key value pair to our store at application deployment time using the command `spin up --key-value`. This feature is helpful when you need to pass in data that's used in the initialization process. If you aren't interested in pre-seeding your key value store, then please use the `spin up` feature as usual. 
```

Comment:
This is a tutorial so, rather than giving me options that I don't know if I need them or not, tell me what you want me to do to make the app I am building work.